Vic2ToHoI4/Source/V2World/World.cpp

```diff
@@ -300,7 +305,7 @@ void Vic2::World::readCountryFiles(const Configuration& theConfiguration)
 	{
 		if (!processCountriesDotTxt(theConfiguration.getVic2Path() + "/common/countries.txt", "", theConfiguration))
 		{
-			LOG(LogLevel::Error) << "Could not open " << theConfiguration.getVic2Path() + "/common/countries.txt";
+			Log(LogLevel::Error) << "Could not open " << theConfiguration.getVic2Path() + "/common/countries.txt";
```

Comment:
throw.

Reply:
also, codacy isn't entiely insane. Instad of mucking about with its errors:
```
	if (const auto countryItr = countries.find(tag); countryItr != countries.end())
	{
		return countryItr->second;
	}
	else
	{
		return std::nullopt;
	}

``` 
You could theoretically do:
```if (!countries.count(tag)) return std::nullopt;
return countries[tag];
```

Reply:
this comment was not supposed to go here, but you get the gist.